.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,201 @@
+        copy /Y /B output\release-static\x86\lib\* %INCLIB%
+
+    - name: Build dependencies / freetype
+      run: |
+        REM Toolkit v100 not available; missing VCTargetsPath; Clean fails
+
+        set INCLUDE=C:\Program Files (x86)\Microsoft SDKs\Windows\V7.1A\Include
+        set INCLIB=%GITHUB_WORKSPACE%\winbuild\depends\msvcr10-x32
+        set BUILD=%GITHUB_WORKSPACE%\winbuild\build
+        cd /D %BUILD%\freetype-2.10.1
+        call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86 8.1
+        echo on
+        rmdir /S /Q objs
+        set DefaultPlatformToolset=v140
+        set VCTargetsPath=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets
+        set MSBUILD="C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\MSBuild.exe"
+        %MSBUILD% builds\windows\vc2010\freetype.sln /t:Build /p:Configuration="Release" /p:Platform=Win32 /m
+        xcopy /Y /E /Q include %INCLIB%
+        copy /Y /B objs\Win32\Release\freetype.dll %INCLIB%
+        copy /Y /B objs\Win32\Release\freetype.lib %INCLIB%
+
+    - name: Build dependencies / lcms2
+      if: false
+      run: |
+        REM Projects\VC2015\lcms2.sln is not available in lcms2-2.7
+
+        set INCLUDE=C:\Program Files (x86)\Microsoft SDKs\Windows\V7.1A\Include
+        set INCLIB=%GITHUB_WORKSPACE%\winbuild\depends\msvcr10-x32
+        set BUILD=%GITHUB_WORKSPACE%\winbuild\build
+        cd /D %BUILD%\lcms2-2.7
+        call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86 8.1
+        echo on
+        rmdir /S /Q Lib
+        rmdir /S /Q Projects\VC2015\Release
+        set VCTargetsPath=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets
+        set MSBUILD="C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\MSBuild.exe"
+        %MSBUILD% Projects\VC2015\lcms2.sln /t:Clean;lcms2_static /p:Configuration="Release" /p:Platform=Win32 /m
+        xcopy /Y /E /Q include %INCLIB%
+        copy /Y /B Lib\MS\*.lib %INCLIB%
+
+    - name: Build dependencies / openjpeg
+      run: |
+        set INCLUDE=C:\Program Files (x86)\Microsoft SDKs\Windows\V7.1A\Include
+        set INCLIB=%GITHUB_WORKSPACE%\winbuild\depends\msvcr10-x32
+        set BUILD=%GITHUB_WORKSPACE%\winbuild\build
+        cd /D %BUILD%\openjpeg-2.3.1msvcr10-x32
+        call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86 8.1
+        echo on
+        cmake.exe -DBUILD_THIRDPARTY:BOOL=OFF -DBUILD_SHARED_LIBS:BOOL=OFF -DCMAKE_BUILD_TYPE=Release -G "NMake Makefiles" .
+        nmake -nologo -f Makefile clean
+        nmake -nologo -f Makefile
+        mkdir %INCLIB%\openjpeg-2.3.1
+        copy /Y /B src\lib\openjp2\*.h %INCLIB%\openjpeg-2.3.1
+        copy /Y /B bin\*.lib %INCLIB%
+
+    - name: Build dependencies / ghostscript
+      if: false
+      run: |
+        REM only used for Python 2.7 in AppVeyor?
```

Comment:
What makes you think that we're currently only using Ghostscript in Python 2.7?

Reply:
https://github.com/python-pillow/Pillow/blob/1608a15ab5e18b96e0145e66c5218e4adee3fbc0/winbuild/build_dep.py#L323
But looking at AppVeyor logs, GhostScript is available in tests.

Reply:
I figured it out: `C:\Python27\` is used because it is included in `%PATH%`.